src/app/components/core/asset-readings/readings-graph/readings-graph.component.ts

```diff
@@ -622,7 +622,7 @@ export class ReadingsGraphComponent implements OnDestroy {
           type: 'time',
           time: {
             unit: 'second',
-            tooltipFormat: 'HH:mm:ss:SSS',
+            tooltipFormat: 'YYYY-MM-DD HH:mm:ss.SSS',
             displayFormats: {
               unit: 'second',
               second: 'HH:mm:ss'
```

Comment:
what is the use of displayFormats second item? if this is for X axis then for > 8 hours should we show YYYY-MM-DD as well?

Reply:
We will add a new Jira for this for >1 day data.